integration/tests/cypress_e2e.sh

```diff
@@ -6,6 +6,16 @@ test_name=":cypress: e2e tests"
 test_deploy_inspec_profiles=()
 test_skip_diagnostics=true
 
+do_create_config() {
+    do_create_config_default
+
+    #shellcheck disable=SC2154
+    cat <<EOF >> "$test_config_path"
+[auth_z.v1.sys.service]
+  project_limit = 50
```

Comment:
I went to `50` bc that's what we're doing elsewhere 🤷‍♂

Reply:
The current tests only use 7, so 50 will work!

Reply:
Not sure if it matters if we just bump it up to what we need 🤔 probably better to just give ourselves more leeway